src/crawlers/browser_crawler.js

```diff
@@ -388,7 +387,7 @@ class BrowserCrawler extends BasicCrawler {
     _enhanceCrawlingContextWithPageInfo(crawlingContext, page) {
         crawlingContext.page = page;
 
-        // This is the wierd spam because of browser to proxy not page to proxy.
+        // This is the switch because of browser to proxy not page to proxy.
```

Comment:
I remember commenting on this. Don't remember what I said. But I still don't think it's very clear. Only the two of us know what 'browser to proxy not page to proxy' means. And only until we forget.